docs/xmso.md

```diff
@@ -24,12 +24,13 @@
 
 # -Xmso
 
-
-Sets the initial stack size for operating system threads.
+Sets the native stack size for operating system threads.
 
 You can use the `-verbose:sizes` option to find out the values that the VM is currently using.
 
-If you exceed the maximum value, a `java/lang/StackOverflowError` message is reported.
+When your application makes a JNI call, the VM calculates whether the memory required for the call will exceed the `-Xmso` value. If so, a `java/lang/StackOverflowError` error is thrown.
```

Comment:
I think this needs more clarification. The memory requirements for the single call to JNI are calculated, but not what occurs inside the JNI call.

Reply:
Dan reviewed in https://github.com/eclipse/openj9-docs/issues/247, but I'm happy to add any more clarification that's required (as long as someone explains it to me :) )

Reply:
This isn't accurate.  We only check the stack size on call into the VM not on JNI call out.  We estimate whether the native stack is close to overflow on the callin and use that to determine if we should throw native stackoverflow.

We don't check when making a JNI call to avoid the performance cost.

Reply:
I could change it to: 

> When a native method makes a call into the VM, the VM estimates whether the native stack is close to overflowing. If the native method call is likely to overflow the native stack, the VM throws a java/lang/StackOverflowError.

(Assuming we really mean "estimates" here, as opposed to "calculates", ie it's a bit of a guess). However, there's no reference to -Xmso. What part does that value actually play?

Reply:
If the native method call is likely to overflow the native stack, `as set by -Xmso`, the VM throws a java/lang/StackOverflowError.